packages/web3torrent/src/components/torrent-info/peer-network-stats/PeerNetworkStats.tsx

```diff
@@ -7,14 +7,19 @@ export type PeerNetworkStatsProps = {torrent: TorrentUI};
 
 const PeerNetworkStats: React.FC<PeerNetworkStatsProps> = ({torrent}: PeerNetworkStatsProps) => {
   const showTimeRemaining = !torrent.originalSeed && torrent.parsedTimeRemaining;
+  const numChannels = torrent.wires.filter(
+    wire =>
+      wire.paidStreamingExtension.leechingChannelId || wire.paidStreamingExtension.seedingChannelId
+  ).length;
   return (
     !IdleStatuses.includes(torrent.status) && (
       <section className="PeerNetworkStats">
         {showTimeRemaining && `${torrent.parsedTimeRemaining}. `}
         {prettier(torrent.done || !torrent.downloadSpeed ? 0 : torrent.downloadSpeed)}
         /s down, {prettier(!torrent.uploadSpeed ? 0 : torrent.uploadSpeed)}/s up
         <br />
-        <strong data-test-selector="numPeers">{torrent.numPeers}</strong> Peers connected
+        <strong data-test-selector="numPeers">{numChannels}</strong> running channel
```

Comment:
```suggestion
        <strong data-test-selector="numPeers">{numChannels}</strong> running channels
```

Reply:
There is a statement after this checking for `=== 1` and add an `s` if false

Reply:
Gotcha, sorry I missed it.